contracts/hardhat.config.ts

```diff
@@ -18,18 +19,41 @@ const ALCHEMY_KEY = process.env.ALCHEMY_KEY || "";
 const SEQUENCER_PRIVATE_KEY = process.env.SEQUENCER_PRIVATE_KEY;
 const ETHERSCAN_API_KEY = process.env.ETHERSCAN_API_KEY || "";
 const SEQUENCER_ADDRESS = process.env.SEQUENCER_ADDRESS ?? wallet.address;
+const DEPLOYER_PRIVATE_KEY = process.env.DEPLOYER_PRIVATE_KEY;
+const DEPLOYER_ADDRESS = process.env.DEPLOYER_ADDRESS ?? wallet.address;
 
 function createConfig(network: string) {
```

Comment:
need to add some documentation here about what these different networks mean, it's a bit confusing